PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,9 @@
+Fixes # .
+
+
+Changes proposed in this pull request:
+-
+-
+-
+
+@teammodding/full-members
```

Comment:
What's this line for?

Reply:
I'm assuming you mean the bottom one. It should ping all full members of the TEAMModding organization so they know to look at the pull request.

Reply:
People already get notified if they set up the notifications. Do we need it to also ping everyone?

Reply:
No, but the GitHub template suggested it. I'll remove it.